docs/images/logo-title.svg

```diff
@@ -0,0 +1,17 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="762" width="548">
```

Comment:
Should we use your new logo instead of https://github.com/kubeflow/katib/blob/master/docs/images/Katib_Logo.png to be consistent ?
WDYT @gaocegege @johnugeorge ?